introduction.tex

```diff
@@ -20,22 +20,71 @@ \chapter{Introduction}
 
 \section{Terminology}
 
-A \emph{platform} is a single integrated circuit consisting of one or more
-\emph{components}. Some components may be RISC-V cores, while others may have a different
-function. Typically they will all be connected to a single system bus.
-A single RISC-V core contains one or more hardware threads, called
-\emph{harts}.
-
-\emph{DXLEN} of a hart is its widest supported XLEN, ignoring the current value
-of \Fmxl in \Rmisa.
-
-A \emph{physical address} is directly usable on the system bus.
-
-A \emph{virtual address} is an address as a hart sees it. If there is address
-translation this may be different from the \emph{physical address}. If there is
-no translation then it will be the same.
-
-\subsection{Context}
+\begin{description}[style=nextline]
+    \item[AMO]
+        Atomic Memory Operation.
+    \item[BIST]
+        Built-In Self Test.
+    \item[BYPASS]
+        JTAG instruction that selects a single bit data register, also called BYPASS.
+    \item[component]
+        A RISC-V core, or other part of a platform.
+        Typically all components will all be connected to a single system
+        bus.
+    \item[CSR]
+        Control Status Register.
+    \item[DM]
+        Debug Module.
+    \item[DMI]
+        Debug Module Interface.
+    \item[DR]
+        Data Register.
+    \item[DTM]
+        Debug Transport Module.
+    \item[DXLEN]
+        Debug XLEN, which is the widest XLEN a hart supports, ignoring the
+        current value of \Fmxl in \Rmisa.
+    \item[GPR]
+        General Purpose Register.
+    \item[hart]
+        A hardware thread in a RISC-V core.
+    \item[IDCODE]
+        32-bit Identification CODE, and a JTAG instruction that returns the IDCODE value.
+    \item[IR]
+        Instruction Register.
+    \item[JTAG]
+        Joint Test Action Group.
+    \item[MIE]
+        Machine Interrupt Enable register; also a bit in mstatus.
```

Comment:
```suggestion
        Machine-Mode Interrupt Enable register; also a bit in mstatus.
```

When using `MIE` do you always mean the bit in `mstatus`? Would you use `mie` if you were talking about the actual register?

Reply:
I agree on the distinction between MIE and mie.  (I find it confusing but I think it's too late now.)

Reply:
The privspec defines this register as "Machine interrupt-enable register." so -mode seems out of place. In this spec, we always style mie lowercase, and never refer to the bit in mstatus.